src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,97 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize: $pf-global--FontSize--sm;
+
+// label
+$pf-chart-global-label--FontSize: $pf-chart-global--FontSize;
+$pf-chart-global-label--Padding: 10px;
+
+// Layout Props
+$pf-chart-global-layout--Padding: 50px;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
```

Comment:
@mcoker  im not sure whether to break stroke width into two words or not

Reply:
I think this is fine, though I think we should follow the same general formula we do for other [global vars](https://pf4.patternfly.org/guidelines) and use `$pf-chart-area--stroke--Width`. Though what is the "stroke" width? Is that the same as a border?

Reply:
Thats the thing I was debating. Stroke width is different from border width, so I thought it was more of a concept rather than a property. What do you think @mcoker

Reply:
i.e. here there is a stroke width and a border width?

<img width="273" alt="Screen Shot 2019-05-30 at 11 27 41 AM" src="https://user-images.githubusercontent.com/20118816/58643779-f6152c00-82cd-11e9-9d9c-f3ef63990cd9.png">

Reply:
Yes, they are different.